apps/cookbook/src/app/examples/dropdown-example/examples/dropdown-right-aligned.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-dropdown-example-dropdown-right-aligned',
```

Comment:
Consider naming:
```suggestion
  selector: 'cookbook-dropdown-example-direction',
```
OR
```suggestion
  selector: 'cookbook-dropdown-example-open-left',
```

Reply:
I like the `open-left` - it's most descriptive 👍